generators/gae/templates/application-prod-gae.yml.ejs

```diff
@@ -27,23 +27,26 @@
 # Full reference is available at:
 # http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
 # ===================================================================
-
+server:
+    port: ${PORT}
+<%_ if (this.gaeCloudSQLInstanceNeeded === 'Y') { _%>
```

Comment:
`this.gaeCloudSQLInstanceNeeded`  -> `gaeCloudSQLInstanceNeeded`

Reply:
Thanks. Done :smile: